core/src/main/java/io/kestra/core/models/tasks/runners/PluginUtilsService.java

```diff
@@ -186,10 +185,12 @@ public static Map<String, Object> parseOut(String line, Logger logger, RunContex
                 if (bashCommand.getLogs() != null) {
                     bashCommand.getLogs().forEach(logLine -> {
                         try {
+                            if (customInstant != null) {
+                                runContext.setRunContextLoggerTimestamp(customInstant);
+                            }
```

Comment:
This is not a good idea, the timestamp must be passed inside the event not globally inside the logger, see my other comment for explanation.
For that, we may switch to the logback API instead of the SLF4J one.